lms/templates/financial-assistance/financial_assessment_form.underscore

```diff
@@ -39,9 +39,9 @@
 	<%= HtmlUtils.HTML(fieldsHtml) %>
 
 	<div class="cta-wrapper clearfix">
-		<a href="<%- student_faq_url %>" class="nav-link"><%- interpolate_text(
-			gettext('Back to {platform} FAQs'),
-    		{platform: platform_name}
+		<a href="<%- student_faq_url %>" class="nav-link"><%- edx.HtmlUtils.interpolateHtml(
```

Comment:
Nit: You'd don't need to use the edX namespace as ``HtmlUtils.template`` injects ``HtmlUtils`` in as a context variable. If that doesn't work, it means you are using ``_.template`` to render it.